automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -280,6 +282,55 @@ trait Orchestration extends FireCloudClient with LazyLogging {
     postRequestWithMultipart(apiUrl(s"api/workspaces/$ns/$wsName/importEntities"), fileName, fileContent)
   }
 
+  object trial {
+
+    case class TrialProjectReport(unverified: Int,
+                                  errored: Int,
+                                  available: Int,
+                                  claimed: Int)
+
+    def enableUser(userEmail: String)(implicit token: AuthToken): Unit = {
+      val enableResponse: String = postRequest(apiUrl("api/trial/manager/enable"), Seq(userEmail))
+      val responseJson: JsObject = enableResponse.parseJson.asJsObject
+      val successfulResponseKeys = Seq("Success", "NoChangeRequired")
+      responseJson.fields.map {
+        case f@x if successfulResponseKeys.contains(f._1) =>
+          logger.info(s"${f._1}: ${f._2.toString()}")
+          return
+        case f@y =>
+          logger.error(s"${f._1}: ${f._2.toString()}")
+          throw new Exception(s"Unable to enable user: $userEmail. Error message: $enableResponse")
+      }
+    }
+
+    def createTrialProjects(count: Int)(implicit token: AuthToken): Unit = {
+      val report: TrialProjectReport = countTrialProjects()(token)
+      if (report.available == 0) {
```

Comment:
Shouldn't this be `if (report.available >= count)` in case there are more than 1 project but less than the number requested?

Reply:
Good catch!